README.md

```diff
@@ -1,5 +1,31 @@
 # RayFed
-A multiple parties joint, distributed execution engine on the top of Ray.
+A multiple parties joint, distributed execution engine based on Ray, to help build your own federated learning frameworks in minutes.
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let you write your federated and distributed computing applications like a single-machine program.
+
+- **Multiple Controller Execution Mode**  
+  
+  The RayFed job can be run in single controller mode when developing and debugging, while it will be run in multiple controller mode in production without any code change.
```

Comment:
My suggestion: "The RayFed job can be run in not only the single-controller mode for developing and debugging but also in the multiple-controller mode for production scenarios without any code change"

Besides, I think a short intro for the concept of "single-" & "multi-" controller is needed. I will follow up on this kinds of contents.

Reply:
Nice suggestion.